src/client/components/ProductDetail/ProductDetail.component.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './ProductDetail.styles.css';
+import Button from '../Button/Button.component';
```

Comment:
Nitpick(import order):
It's often helpful to import things in order of how "far away" they are.
Button is further away (one folder) than the CSS styles, and react is even further way (has to be downloaded). 
Just a small detail.